pkg/url/labels/labels.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // URLLabel is the label key that is applied to all url resources
 // that are created
-const URLLabel = "app.kubernetes.io/url-name"
+const URLLabel = "app.openshift.io/vcs-uri-name"
```

Comment:
I don't see any label change for URL in https://github.com/openshift/odo/issues/599#issuecomment-488610781

Reply:
Good cache. 
This should not have been updated. This label is for odo internal use.

This should probably be something like `odo.openshift.io/url-name` to indicate that it is odo specific. I'll update it